pg_sparse/README.md

```diff
@@ -0,0 +1,133 @@
+# pg_sparse
+
+## Overview
+
+`pg_sparse` is a PostgreSQL extension that enables similarity search over sparse vectors
+in Postgres using HNSW.
+
+## Installation
+
+### From Self-Hosted Postgres
+
+If you are self-hosting Postgres and would like to use the extension within your existing
+Postgres, follow these steps:
+
+1. Install Rust and cargo-pgrx:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+cargo install cargo-pgrx --version 0.9.8
+```
+
+2. Then, run:
+
+```bash
+# Clone the repo (optionally pick a specific version)
+git clone https://github.com/paradedb/paradedb.git --tag <VERSION>
+
+# Install pg_sparse
+cd pg_sparse/
+cargo pgrx init --pg<YOUR-POSTGRES-MAJOR_VERSION>=`which pg_config`
+cargo pgrx install
```

Comment:
When the PR is ready, I can do a pass over the README to add comments about self-hosting it :)

Reply:
You can also just copy-paste the pg_bm25 README and update pg_bm25 to pg_sparse and it'll be correct :)